src/consts.rs

```diff
@@ -1,5 +1,5 @@
 // Expired seconds of cache data.
-pub const DEFAULT_CACHE_EXPIRED_SECS: u64 = 1;
+pub const DEFAULT_CACHE_EXPIRED_SECS: u64 = 15;
```

Comment:
@georgehao and @HAOYUatHZ , could help check if we could set default expired time to `15` seconds? btw we could also set it from ENV with this PR.